main.py

```diff
@@ -4,24 +4,96 @@
 
 # Import config data
 from config import prefix
+from management.db import db_set
+from interpretation.ww_head import process
 import config
 
+
 client = discord.Client()
 
-'''
-To spare some time, the program should keep track of channels that it has sent messages in before.
-This way, sending messages to the right channels should (hopefully) save some time,
-as the bot doesn't need to ask the API for a channel for every message.
-Instead, it should save a table with the channel objects.
-'''
-channel_table = []
 
 # Whenever a message is sent.
 @client.event
 async def on_message(message):
     # we do not want the bot to reply to itself
     if message.author == client.user:
         return
+    
+    # Check if the message author has the Game Master role
+    isGameMaster = False
+    if False:
+        isGameMaster = True
```

Comment:
wtf is this?

Reply:
Check feature_polls for the gm_only thing

Reply:
i know. i wrote that.

Reply:
If the user has the game master role, it's set to True.
It's defined as False beforehand to prevent a ValueError.

Reply:
the funny thing is that that is not the case. `if False` never executes

Reply:
also even if you could just do
```py
isGameMaster = somefunc()
```
also please use snake_case. this is python ffs

Reply:
Yes, it's supposed to have a function. But that function simply does not exist yet, so I decided to set it to False now, to prevent bugs AND to prevent anyone from having Game Master privileges.

Reply:
Use our system and you can use the `game_masters_only()` decorator